python-minimal/README.md

```diff
@@ -0,0 +1,20 @@
+# python-minimal
+
+This package is intended to be used only as a build dependency for glibc.
+
+If you would like to use Python in your applications, please look at `core/python` or one of the other versioned Python packages.
```

Comment:
Maybe people will just see this package on bldr, think a `minimal` version is neat, and try using it... is there something we can do that's more active in stopping people? e.g. have this plan crippled as runtime dependency so it can only ever be used as a build dep?

Reply:
I don't know that there's anything to do here.  Perhaps this is a feature request for Builder, to expose the README or `pkg_description` on the package page?  That would still required clicking through into the package page, and that still doesn't cover  hab pkg install. 

If we encounter usage in the wild and get questions, I think we'll need to direct people to the README and one of the other python packages, and gently explain that usage of this package is unsupported outside of building glibc.